ui/components/app/flask/snap-ui-markdown/snap-ui-markdown.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+import { TYPOGRAPHY } from '../../../../helpers/constants/design-system';
+import Typography from '../../../ui/typography/typography';
+
+export const SnapUIMarkdown = ({ children }) => {
+  return (
+    <Typography variant={TYPOGRAPHY.H6}>
+      <ReactMarkdown
+        className="snap-ui-markdown__text"
+        allowedElements={['p', 'strong', 'em']}
+        components={{ p: Fragment }}
+      >
+        {children}
+      </ReactMarkdown>
+    </Typography>
```

Comment:
We shouldn't wrap all markdown in a paragraph (`Typography`).

```suggestion
    <ReactMarkdown
      className="snap-ui-markdown__text"
      allowedElements={['p', 'strong', 'em']}
      components={{ p: Typography /* and something to set the variant */ }}
    >
      {children}
    </ReactMarkdown>
```

Reply:
Fixed 👍